examples/threathunter/threat_intelligence/stix_parse.py

```diff
@@ -181,15 +182,23 @@ def cybox_parse_observable(observable, indicator, timestamp, score):
             link = reference
```

Comment:
Are these two lines suppose to be indented that far in?? Just wondering, looks inconsistent with the line above

`description = str(indicator.title)`

Reply:
If you expand all the context it looks right.

Reply:
My eyeballs are not working correctly. I retract my statement! Nice catch Amy